packages/components/psammead-brand/src/index.test.jsx

```diff
@@ -165,6 +165,34 @@ describe('Brand', () => {
       expect(document.activeElement).toBe(brand);
     });
 
+    it('should let the SvgWrapper be focussed with a ref if no url prop', () => {
+      const TestComponent = () => {
+        const brandRef = useRef(null);
+
+        useEffect(() => {
+          brandRef.current?.focus();
+        });
```

Comment:
To reflect real-world usage as closely as possible in this test, have you thought about adding an empty dependency array to this hook so that focus is only pulled on mount/unmount?

Reply:
Thanks for all the help and suggestions @simonsinclair 🙌 

I thought about adding it but couldn't think of a reason, but that definitely makes sense, I'll do that now 👍